src/Microsoft.OData.Core/Json/DefaultJsonWriterFactory.cs

```diff
@@ -14,22 +14,23 @@ namespace Microsoft.OData.Json
     /// </summary>
     public sealed class DefaultJsonWriterFactory : IJsonWriterFactory
     {
-        private ODataStringEscapeOption _stringEscapeOption;
+        private ODataStringEscapeOption stringEscapeOption;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="DefaultJsonWriterFactory" />.
         /// </summary>
         public DefaultJsonWriterFactory()
             : this(ODataStringEscapeOption.EscapeNonAscii)
-        { }
+        {
+        }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="DefaultJsonWriterFactory" />.
         /// </summary>
         /// <param name="stringEscapeOption">The string escape option.</param>
         public DefaultJsonWriterFactory(ODataStringEscapeOption stringEscapeOption)
         {
-            _stringEscapeOption = stringEscapeOption;
+            this.stringEscapeOption = stringEscapeOption;

```

Comment:
Some stacks prefer to use underscore for private members. If we don't use this convention then please fix all these occurrences as well - 
ExpressionNormalizer.cs(55), AuthorizationVocabularyTests.cs(23), ValidationVocabularyTests.cs(23), 
ApplyBinderTests.cs(22):        
ApplyBinderTests.cs(24):       
ApplyBinderTests.cs(372)

Reply:
#Resolved.